systemtest/src/test/java/io/strimzi/systemtest/ConnectClusterIT.java

```diff
@@ -113,6 +125,40 @@ public void testDeployUndeploy() {
         testDockerImagesForKafkaConnect();
     }
 
+    @Test
+    @JUnitGroup(name = "regression")
+    @ConnectS2ICluster(name = CONNECT_CLUSTER_NAME, connectConfig = CONNECT_CONFIG_CONVERTER_SCHEMAS_DISABLED)
+    public void testDeployS2IWithMongoDBPlugin() {
+        String pathToDebeziumMongodb = "https://repo1.maven.org/maven2/io/debezium/debezium-connector-mongodb/0.3.0/debezium-connector-mongodb-0.3.0-plugin.tar.gz";
+        // Create directory for plugin
+        kubeClient.execInKubeWorkspace("mkdir", "-p", "./my-plugins/");
+        // Download and unzip MongoDB plugin
+        kubeClient.execInKubeWorkspace("wget", "-O", "debezium-connector-mongodb-plugin.tar.gz", "-P", "./my-plugins/", pathToDebeziumMongodb);
+        kubeClient.execInKubeWorkspace("tar", "xf", "debezium-connector-mongodb-plugin.tar.gz", "-C", "./my-plugins/");
+        // Start a new image build using the plugins directory
+        kubeClient.execInKubeWorkspace("oc", "start-build", "my-cluster-connect", "--from-dir", "./my-plugins/");
+        // Wait 60 secs for message about adding MongoDB plugin to cluster connect
+        waitFor("Wait message in pod log", 5000, 60000,
+            () -> !kubeClient.searchInLog("deploymentConfig", "my-cluster-connect", stopwatch.runtime(SECONDS), "\"Added plugin \'io.debezium.connector.mongodb.MongoDbConnector\'\"").isEmpty());
```

Comment:
In principle this should work. But it relies on the log message. Why not use the REST endpoint `http://localhost:8083/connector-plugins` to list that the connector has been added? Would that be too complicated?